string name # Name of the landmark
float64 distance # Distance to the lanmard, in meter
